BZ-2004124: Adding details on configuring descheduler settings in the UI (#38813)
openshift/openshift-docs: modules/nodes-descheduler-installing.adoc

. Create a descheduler instance.
.. From the *Operators* -> *Installed Operators* page, click the *Kube Descheduler Operator*.
.. Select the *Kube Descheduler* tab and click *Create KubeDescheduler*.
.. Edit the settings as necessary.
... Expand the *Profiles* section to select one or more profiles to enable. The `AffinityAndTaints` profile is enabled by default. Click *Add Profile* to select additional profiles.
+
[NOTE]
====

> Do not enable both `TopologyAndDuplicates` and `SoftTopologyAndDuplicates` because they will conflict with each other.

Reviewer: ```suggestion
Do not enable both `TopologyAndDuplicates` and `SoftTopologyAndDuplicates`. Enabling both results in a conflict.
```

Just a suggestion, I was struggling with the use of "they". Not required to change it though.

Author: Great suggestion! I use this phrasing in another module, and will change it there as well :)
